components/chef-automate-collect/commands/init_cli.go

```diff
@@ -5,11 +5,12 @@ import (
 
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
+	"github.com/chef/chef-workstation/components/main-chef-wrapper/dist"
 )
 
 func newRootCmd() *cobra.Command {
 	r := &cobra.Command{
-		Use:   "chef-automate-collect",
+		Use:   dist.AutomateCollectExec,
 		Short: "Collect data for Chef Automate",
```

Comment:
Should the Short text here also do this?

Reply:
I haven't seen Short done as far as golang is concerned.  Command entry points and websites are in there but not command text.